compiler/src/main/scala/org/scalajs/nscplugin/PrepJSExports.scala

```diff
@@ -531,9 +516,28 @@ trait PrepJSExports[G <: Global with Singleton] { this: PrepJSInterop[G] =>
     }
 
     // Construct proxied function call
-    val sel = gen.mkAttributedRef(trgSym)
+    val sel = {
+      if (trgSym.isConstructor) {
+        Select(New(TypeTree(trgSym.owner.tpe)), trgSym)
```

Comment:
This won't produce valid code when the class is polymorphic. We should use type parameters to the proxy matching the type parameters of the class, and use them in the `New(TypeTree(_))`. We should have a test with such an exported polymorphic member class.